doc/deployment.rst

```diff
@@ -53,7 +53,7 @@ Start by installing ``mod_wsgi``::
 
 Then configure apache::
 
- sudo vim /etc/httd/conf.d/nuancier.conf
+ sudo vim /etc/httpd/conf.d/nuancier.conf
```

Comment:
@pypingou This was the only adjustment that I could find. Other steps went through smoothly.